vernac/comAssumption.mli

```diff
@@ -43,6 +43,18 @@ val declare_variable
   -> variable
   -> GlobRef.t * UVars.Instance.t
 
+val declare_local
```

Comment:
Please document this function and its parameters, maybe also add some more arguments names?

Reply:
OK, will do (e.g. I could use the labels `impargs`, `impl` and `univs` as in `declare.ml`).

Also, do you think `declare_local` could/should better go in `declare.ml`? Or a part of it. For instance, in other `declare_foo`, `assumption_message`/`definition_message` and `maybe_declare_manual_implicits` are done in `declare.ml` so maybe should they be done here also in `declare.ml`?

Reply:
Sure, Hugo, whatever you think is OK; even a very simple note explaining the parameters would suffice IMHO.

Maybe it is only my problem, but I do get a bit confused when functions have so many arguments and I don't have a name for each of them to figure out what they do.